docs/features/logforwarding/outputs/azure-monitor-log-forwarding.adoc

```diff
@@ -0,0 +1,157 @@
+=== Steps to forward to Azure Monitor Log via HTTP Data Collector API
+
+(https://learn.microsoft.com/en-us/azure/azure-monitor/logs/data-collector-api?tabs=powershell)
+
+. Create a secret containing your `shared_key` using the following command (primary or the secondary key for the workspace that's making the request):
++
+----
+ oc create secret generic azure-secret -n openshift-logging --from-literal=shared_key='<shared_key>'
+----
+
+. Create a Cluster Logging instance with collection `type: vector`:
++
+----
+ oc apply -f cluster-logging.yaml
+----
++
+.cluster-logging.yaml
+[source,yaml]
+----
+kind: ClusterLogging
+apiVersion: logging.openshift.io/v1
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  collection:
+    type: vector
+----
+
+. Create a Cluster Log Forwarder instance by specifying your `customerId`, `logType` and the `secret` name:
++
+----
+ oc apply -f cluster-log-forwarder.yaml
+----
++
+.cluster-log-forwarder.yaml
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogForwarder"
+spec:
+  outputs:
+  - name: azureLogAnalyticsApp
```

Comment:
Same nit as I mentioned with the CEE slides.  Match case with our other docs, kebab-case for the values of names.  Low priority if you agree.